webui/react/src/components/NavigationSideBar.tsx

```diff
@@ -110,7 +111,9 @@ const NavigationSideBar: React.FC = () => {
   // `nodeRef` padding is required for CSSTransition to work with React.StrictMode.
   const nodeRef = useRef(null);
 
-  const { auth, resourcePools, info, pinnedWorkspaces } = useStore();
+  const { auth, info, pinnedWorkspaces } = useStore();
+  const { resourcePools: loadableResourcePools } = useResourcePools();
+  const resourcePools = Loadable.getOrElse([], loadableResourcePools);
```

Comment:
You should add a TODO here if you aren't going to properly handle the loading state in this PR.